ai.php

```diff
@@ -12,7 +12,7 @@
  * Plugin URI:        https://github.com/WordPress/ai
  * Description:       AI experiments and capabilities for WordPress.
  * Version:           0.1.0
- * Requires at least: 6.8
+ * Requires at least: 6.9
```

Comment:
Is the reason for requiring 6.9 because of the Abilities API that ships there? We are bundling that right now in this plugin so things work fine on 6.8 (or presumably older versions). I'm fine with this at 6.9 but curious if there's a specific reason for enforcing this?

Reply:
It's a good question. Personally, I'd rather depend on 6.9 and not bundle the Abilities API. I don't think the extra complexity is worth it.

Reply:
Updated in [838649c](https://github.com/WordPress/ai/pull/106/commits/838649cd3809dcae7b13b8953b33b0377c1782fc).

Reply:
> Personally, I'd rather depend on 6.9 and not bundle the Abilities API. I don't think the extra complexity is worth it.

I concur, but I'll open a follow-up issue for this and we can change in v0.2.0.

Reply:
In which case we'd need to set Gutenberg as a plugin dependency, which while I'm in favor of Gutenberg being used I don't know that I want AIE to require a plugin dependency.  Unless what you're saying is that we'd leverage an npm package from Gutenberg instead of from WP core?